src/storage/src/render/sources.rs

```diff
@@ -333,6 +335,8 @@ where
                                 persist_clients,
                                 description.storage_metadata.clone(),
                                 Antichain::from_elem(previous_as_of),
+                                Antichain::new(),
```

Comment:
I believe this one could use the `previous_as_of` as `until`, but I would double check with @brennanvincent. But I think it's fine to merge as is.

Reply:
Yes, I'm not sure what these are meant to be, and didn't want to muck up someone else's work guessing (nor pressure them with threats of mucking it up if they don't read it quickly). I was going to file an issue instead!

Reply:
Ah, quick sanity check: I think it cannot use `previous_as_of`, because anything greater *or equal to* the frontier will be suppressed. I think we might want "just after `previous_as_of`".